vip-keycloak.postman_collection.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Transfer it in `vip-api/src/test/resources`

Reply:
done